src/providers/analytics/provider.test.ts

```diff
@@ -69,7 +69,7 @@ describe("IndexAnalyticsProvider", () => {
     expect(analyticsData.symbol).toEqual("MVI")
     expect(analyticsData.decimals).toEqual(18)
     expect(analyticsData.marketPrice).toEqual(marketPrice)
-    expect(analyticsData.navPrice).toEqual(navPrice)
+    expect(analyticsData.navPrice).toBeCloseTo(navPrice)
```

Comment:
In more volatile markets it otherwise would fail for lil differences in the 0.0000000001 region.